homeassistant/components/spotify/media_player.py

```diff
@@ -378,6 +399,15 @@ def update(self) -> None:
         current = self._spotify.current_playback()
         self._currently_playing = current or {}
 
+        item = self._currently_playing.get("item") or {}
+        if item is not None:
+            self._state_attributes["audio_features"] = {}
+            uri = item.get("uri")
+            audio_features = self._spotify.audio_features(uri)[0]
+            for feature, value in audio_features.items():
+                if feature in AUDIO_FEATURE_LIST:
+                    self._state_attributes["audio_features"][feature] = value
```

Comment:
I am not sure I follow what you mean regarding multiple accounts and corresponding config entries. This may be just be my lack of understanding of home assistants inner workings.

In the situation where there are multiple spotify accounts configured would there not be a corresponding media player entity for each? Then the audio feature attributes are unique to that account and corresponding entity?

The idea with the current implementation would be to use the template sensors to grab the state information. Then users can create the sensors they are interested in rather then have several default sensors.

Reply:
Each config entry (Spotify account) would create those sensors.

> Then users can create the sensors they are interested in rather then have several default sensors.

That is not a valid reason for the current structure. We provide a default enabled/disabled on entities. For less common entities we disable them by default. So they are created, but not in use until activated by the user in the UI. This way, one can use them without any configuration but activate with a single click.